types/parcel-bundler/index.d.ts

```diff
@@ -173,6 +184,23 @@ declare class ParcelBundler {
     addPackager(type: string, packager: string): void;
 
     bundle(): Promise<ParcelBundler.ParcelBundle>;
+
+    /**
+     * Starts a simple http (or https) server.
+     *
+     * @param `https` `{ cert, key }` pointing to the location of key and cert file or `true` to generate a key
+     */
+    serve(port?: number, https?: false, host?: string): Promise<http.Server>;
+    serve(
+        port: number,
+        https: true | HttpsInfo,
+        host?: string
+    ): Promise<https.Server>;
+    serve(
+        port?: number,
+        https?: boolean | HttpsInfo,
+        host?: string
+    ): Promise<http.Server | https.Server>;
```

Comment:
The 2nd overload doesn't have optionals for the first 2 parameters but I think the JS library allows them to be optional. If this is in fact the case, this function can be written as:

```ts
serve<T extends boolean | HttpsInfo>(sort?: number, https?: T, host?: string): Promise<T extends false ? http.Server : https.Server>
```